### DISCUSSION TITLE
Icons in the fancy tab bar ("X" and "+") look wierd?

### DISCUSSION THREAD

**Discussion by kskruffnekk** [wezterm/wezterm#4241] [Q&A]
Hello there,
as seen in the screenshots, the font rendering especially in the tab-section looks very wierd.

I tried JetBrains Mono, FiraCode Nerd, Iosevka. All of them deliver the same problem. What am I doing wrong?

It doesn't matter which option I choose with the freetype-settings...

The + for new tab looks cursed, what is that?

![Bildschirmfoto vom 2023-09-03 20-15-50](https://github.com/wez/wezterm/assets/108282882/9da6e08c-3772-4d4b-8ea5-ffc81f93d87b)
![Bildschirmfoto vom 2023-09-03 20-28-14](https://github.com/wez/wezterm/assets/108282882/fa1aed0f-d5e8-4ef6-b0ce-539ee200c214)

`return {
  -- option	= value	, [default] comment
  front_end = "WebGpu",
  enable_wayland = true,
  initial_cols = 130,   -- Width
  initial_rows = 40,    -- Height
  window_close_confirmation = 'NeverPrompt',  -- No prompt when closing terminal ['AlwaysPrompt']
  default_cursor_style = 'BlinkingUnderline',
  tab_bar_at_bottom = false,
  dpi = 96.0,
  bold_brightens_ansi_colors = true,
  freetype_load_target = "Light",
  freetype_load_flags = "FORCE_AUTOHINT",

  -- Fonts
  font     	= wezterm.font('JetBrains Mono', {weight = 'Regular', italic = false }), -- [JetBrains Mono]
  -- font  	= wezterm.font("JetBrains Mono", {weight="Bold",italic=true})	, -- [JetBrains Mono] Name with parameters
  font_size	= 12.0                                                       	, -- [12.0]

  -- Colors
  color_scheme     	= "Monokai Soda", -- full list @ wezfurlong.org/wezterm/colorschemes/index.html
  colors           	= {        	  -- color_scheme takes precedence over these
    foreground     	= "white" 	, -- [silver] The default text color
    background     	= "black"  	, -- [black]  The default background color
    cursor_bg      	= "#52ad70"	, -- [#52ad70] Overrides the cell background color when the current cell is occupied by the cursor and the cursor style is set to Block
    cursor_fg      	= "black"  	, -- [black]   Overrides the text color when the current cell is occupied by the cursor
    cursor_border  	= "#52ad70"	, -- [#52ad70] Specifies the border color of the cursor when the cursor style is set to Block, of the color of the vertical or horizontal bar when the cursor style is set to Bar or Underline.
    selection_fg   	= "black"  	, -- [black]   The foreground color of selected text
    selection_bg   	= "#fffacd"	, -- [#fffacd] The background color of selected text
    scrollbar_thumb	= "#222222"	, -- [#222222] The color of the scrollbar "thumb"; the portion that represents the current viewport
    split          	= "white"	, -- [#1A1A1A] The color of the split lines between panes
  --   ansi        	= {"black", "maroon", "green", "olive", "navy", "purple", "teal", "silver"},
  --   brights     	= {"grey", "red", "lime", "yellow", "blue", "fuchsia", "aqua", "white"},
  --   indexed     	= {[136]="#af8700"} , -- Arbitrary colors of the palette in the range from 16 to 255
  },

 -- Appearance
  window_background_opacity   	= 0.9  	, -- [1.0] alpha channel value with floating point numbers in the range 0.0 (meaning completely translucent/transparent) through to 1.0 (meaning completely opaque)
  enable_tab_bar              	= true	, -- [true]
  hide_tab_bar_if_only_one_tab	= false	, -- [false] hide the tab bar when there is only a single tab in the window
  use_fancy_tab_bar             = true  ,
    
  window_frame = {
    -- The font used in the tab bar.
    -- Roboto Bold is the default; this font is bundled
    -- with wezterm.
    -- Whatever font is selected here, it will have the
    -- main font setting appended to it to pick up any
    -- fallback fonts you may have used there.
    font = wezterm.font('JetBrains Mono', {weight = 'Regular', italic = false }),
    inactive_titlebar_border_bottom = '#1A1A1A',  -- wierd [purple] line in master config
    active_titlebar_border_bottom = '#1A1A1A',    -- wierd [purple] line in master config
    inactive_titlebar_fg = '#cccccc',
    active_titlebar_fg = '#ffffff',
    -- Minimize / Maximize / Close - Buttons
    button_fg = 'white',
    button_bg = '#1A1A1A',
    button_hover_fg = 'white',
    button_hover_bg = 'black',
  
    -- The size of the font in the tab bar.
    -- Default to 10.0 on Windows but 12.0 on other systems
    font_size = 12.0,
      -- The overall background color of the tab bar when
    -- the window is focused
    active_titlebar_bg = '#1A1A1A',

    -- The overall background color of the tab bar when
    -- the window is not focused
    inactive_titlebar_bg = '#1A1A1A',
  },`

**Comment by Shimoon** [comment]
Up this one!
Especially the "+" sign which look weird.
